modules/cluster-logging-deploy-cli.adoc

```diff
@@ -373,31 +282,68 @@ elasticsearch-cdm-x6kdekli-3   1/1     1            0           6m44s
 +
 [source,terminal]
 ----
-$ oc create -f <file-name>.yaml
+$ oc apply -f <filename>
 ----
 +
-For example:
+This creates the {logging} components, the `Elasticsearch` CR and components, and the Kibana interface.
+
+.Verification
+
+. Verify the {es-op} installation:
 +
 [source,terminal]
 ----
-$ oc create -f olo-instance.yaml
+$ oc get csv --all-namespaces
 ----
 +
-This creates the {logging} components, the `Elasticsearch` custom resource and components, and the Kibana interface.
+.Example output
+[source,terminal]
+----
+NAMESPACE                                               NAME                                            DISPLAY                  VERSION               REPLACES   PHASE
+default                                                 elasticsearch-operator.5.1.0-202007012112.p0    OpenShift Elasticsearch Operator   5.5.0-202007012112.p0               Succeeded
+kube-node-lease                                         elasticsearch-operator.5.5.0-202007012112.p0    OpenShift Elasticsearch Operator   5.5.0-202007012112.p0               Succeeded
+kube-public                                             elasticsearch-operator.5.5.0-202007012112.p0    OpenShift Elasticsearch Operator   5.5.0-202007012112.p0               Succeeded
+kube-system                                             elasticsearch-operator.5.5.0-202007012112.p0    OpenShift Elasticsearch Operator   5.5.0-202007012112.p0               Succeeded
+openshift-apiserver-operator                            elasticsearch-operator.5.5.0-202007012112.p0    OpenShift Elasticsearch Operator   5.5.0-202007012112.p0               Succeeded
+openshift-apiserver                                     elasticsearch-operator.5.5.0-202007012112.p0    OpenShift Elasticsearch Operator   5.5.0-202007012112.p0               Succeeded
+openshift-authentication-operator                       elasticsearch-operator.5.5.0-202007012112.p0    OpenShift Elasticsearch Operator   5.5.0-202007012112.p0               Succeeded
+openshift-authentication                                elasticsearch-operator.5.5.0-202007012112.p0    OpenShift Elasticsearch Operator   5.5.0-202007012112.p0               Succeeded
+...
+----
++
+There should be an {es-op} instance in each namespace. The version number might be different than shown.
 
-. Verify the installation by listing the pods in the *openshift-logging* project.
+. Verify the {clo} installation.
 +
-You should see several pods for components of the Logging subsystem, similar to the following list:
+There should be a Red Hat OpenShift Logging Operator in the `openshift-logging` namespace. The Version number might be different than shown.
++
+[source,terminal]
+----
+$ oc get csv -n openshift-logging
+----
++
+.Example output
+[source,terminal]
+----
+NAMESPACE                                               NAME                                         DISPLAY                  VERSION               REPLACES   PHASE
```

Comment:
```suggestion
NAMESPACE                            NAME                                         DISPLAY                  VERSION               REPLACES   PHASE
```
This one, too. A lot of space between namespace and name, but once we get to Version, things are not aligned.